Resources/Locale/en-US/chat/managers/chat-manager.ftl

```diff
@@ -52,28 +52,65 @@ chat-manager-admin-channel-name = ADMIN
 chat-speech-verb-suffix-exclamation = !
 chat-speech-verb-suffix-exclamation-strong = !!
 chat-speech-verb-suffix-question = ?
+chat-speech-verb-suffix-stutter = -
+chat-speech-verb-suffix-mumble = ..
 
 chat-speech-verb-default = says
 chat-speech-verb-exclamation = exclaims
 chat-speech-verb-exclamation-strong = yells
 chat-speech-verb-question = asks
+chat-speech-verb-stutter = stutters
+chat-speech-verb-mumble = mumbles
+
+chat-speech-verb-basic-1 = speaks
+chat-speech-verb-basic-2 = utters
```

Comment:
remove `speaks` and `utters` i dont like them over just `says`

Reply:
Sure, done.